extensions-core/kafka-extraction-namespace/src/main/java/io/druid/query/lookup/KafkaLookupExtractorFactory.java

```diff
@@ -327,7 +327,7 @@ public boolean close()
   }
 
   @Override
-  public boolean replaces(@Nullable LookupExtractorFactory other)
+  public boolean equals(@Nullable Object other)
```

Comment:
yeah , but we can't have both replaces and version controlled via the wrapper. wrapper approach with explicit versioning is concluded better overall.